exercises/isogram/canonical-data.json

```diff
@@ -124,7 +124,9 @@
     {
       "uuid": "0d0b8644-0a1e-4a31-a432-2b3ee270d847",
       "description": "word with duplicated character and with two hyphens",
-      "comments": ["This test aims to catch buggy implementations that check for duplicate spaces or hyphens."],
+      "comments": [
+        "This test aims to catch buggy implementations that check for duplicate spaces or hyphens."
+      ],
```

Comment:
For consistency it'd make more sense to leave this as a one line comment, I think. The line isn't that long.